src/content/api/loaders.md

```diff
@@ -471,3 +509,96 @@ Hacky access to the Compiler object of webpack.
 ### `this._module`
 
 Hacky access to the Module object being loaded.
+
+
+## Error Reporting
+
+There are some ways to throw out errors from inside a loader:
+
+### Using `this.emitError`
+
+It just report errors but not interrupt module compiling.
+
+see [this.emitError](/api/loaders/#this-emiterror).
+
+### Using `throw` (or other uncaught exception)
+
+Throw out an uncaught error while loader are running will cause current module compiling failed.
+
+For example:
+
+__./src/index.js__
+
+```js
+
+require('./loader!./lib');
+
+```
+
+__./src/loader.js__
+
+```js
+
+module.exports = function(source){
+  throw new Error('Here is an fatal Error!');
+};
+
+```
+
+the module that ouccur this error will be packed into bundle (as an error module) like the following:
+
+```js-with-links
+
+/***/ "./src/loader.js!./src/lib.js":
+/*!************************************!*\
+  !*** ./src/loader.js!./src/lib.js ***!
+  \************************************/
+/*! no static exports found */
+/***/ (function(module, exports) {
+
+throw new Error("Module build failed (from ./src/loader.js):\nError: Here is an fatal Error!\n    at Object.module.exports (/workspace/src/loader.js:3:9)");
+
+/***/ })
+
+```
+
+Then the build output will also display the error (Similar to `this.emitError`):
+
+```bash
+
+ERROR in ./src/lib.js (./src/loader.js!./src/lib.js)
+Module build failed (from ./src/loader.js):
+Error: Here is an fatal Error!
+    at Object.module.exports (/workspace/src/loader.js:2:9)
+ @ ./src/index.js 1:0-25
+
+```
+
+And you can see these informations below, not only error message, but also details about which loader and module are involved:
```

Comment:
`As you can see below` would sound better

Reply:
For sure, shorter and clearer